core/api/package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "tsc-check-noimplicitany": "tsc --noEmit -p tsconfig.no-implicit-any.json",
     "tsc-check": "tsc --noEmit -p tsconfig.d.json && tsc --noEmit",
-    "madge-check": "yarn madge --circular lib/servers/graphql-main-server.js",
+    "madge-check": "yarn madge --circular --extensions ts ./src",
```

Comment:
I think we can also delete `yarn` here right?

Reply:
Confirmed - calling `yarn` from within a `package.json` `script` is redundant - by removing it it becomes compatible with `pnpm`.

Reply:
Yes, I actually did do that in the [other PR](https://github.com/GaloyMoney/galoy/pull/3360/files#diff-96fde044b147bac83eceed2332a32601b6400e185e9034973e0110d25c8175baR6) too